### PR TITLE
support for non-equal environments in all.equal (per r-devel)

### PR DESCRIPTION
**Pull request by jonocarroll**
Per this discussion https://stat.ethz.ch/pipermail/r-devel/2020-December/080172.html an added argument to specify that the environments need not be equal needs to be added. This adds that.

Checked locally on r-devel 2021-01-01 r79748.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
 cache: packages
 
-r_github_packages: 
-- metrumresearchgroup/covrpage   
+apt_packages:
+  - libharfbuzz-dev
+  - libfribidi-dev
+
+r_github_packages:
+- metrumresearchgroup/covrpage
```
Comment (yonicd) [inline .travis.yml:10]: covrpage has moved to yonicd/covrpage

Comment (jonocarroll) [reply .travis.yml:10]: Good point. I should probably be testing that with the CRAN version, though.

Comment (yonicd) [reply .travis.yml:10]: There’s a CRAN version?

Comment (jonocarroll) [reply .travis.yml:10]: Ah, disregard.

**Comment by jonocarroll** [comment]
Accepted to CRAN

**Pull request merged**